xml/System.Threading.Tasks/Task.xml

```diff
@@ -1972,7 +1972,7 @@ End Sub
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- After the specified time delay, the task is completed in <xref:System.Threading.Tasks.TaskStatus?displayProperty=fullName> state.  
+ After the specified time delay, the task is completed in <xref:System.Threading.Tasks.TaskStatus.RanToCompletion> state.  
```

Comment:
The xref should be `<xref:System.Threading.Tasks.TaskStatus.RanToCompletion?displayProperty=fullName>`.